app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/AppCompatPreferenceActivity.java

```diff
@@ -0,0 +1,8 @@
+package com.greenfox.fuchsit.zerdareader.activity;
+
+/**
+ * Created by false on 2017. 02. 07..
+ */
+
+public class AppCompatPreferenceActivity {
```

Comment:
@adamgyulavari sorry, this is a forgotten 'leftover' from one of the solutions I was trying to use. I deleted it and pushed again